bin/docker/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM       fedora:22
+
+# install the required dependencies to compile native extensions
+RUN        dnf -y install gcc-c++ make ruby-devel libxml2-devel libxslt-devel findutils git ruby
+RUN        dnf -y install wget # needed by the website to fetch resources
```

Comment:
That's an interesting debate. I understand the demise of Docker caching but for that specific image maintained by non docker experts and used on the desktop (not in prod or anything), is it a good trade off to collapse these layers? I'm open to collapse but it has a bit of a readability cost.